web/cobrands/sass/_base.scss

```diff
@@ -849,8 +870,14 @@ input.final-submit {
 }
 
 .btn--block {
-    display: block;
-    text-align: center;
+    // !importants to override any .btn / .btn-primary styles defined in cobrands
```

Comment:
Is this needed for a cobrand you're aware of? I can't see any that it would affect from a quick look; presumably they've all been okay without the `!important` so far. Just wondering if it'd be nicer to go without them if possible.

Reply:
I needed it in fixmystreet.com cobrand because `.btn-primary` sets its own `width: auto`, and is defined _after_ `.btn--block`, so takes precedence. Didn’t check any of the other cobrands.

Reply:
Ah I didn't find that one because I searched for `btn--`, odd that one (and btn-danger) is named differently from the rest.
Shall we fix that, then? My idea would be button-reset should itself include two mixins, one for layout, one for colours, and then .com (and lincs and oxon) could include the colour mixin and not affect the layout, and then it'd all be flat?

Reply:
Yes, we started moving towards a bootstrap-style `class="btn btn-modifier"` (or, given our leaning towards BEM, `btn--modifier`) pattern a little while ago, but never completed it. I agree it’d be good to set layout-related stuff in a `.btn` class, and then presentation/colour-related stuff in `.btn-primary`, `.btn-default` etc.

Do you think that should be done as part of this PR though? It could be quite far-reaching. Ideally we’d use it as an opportunity to clean up the classes applied to button elements in every cobrand’s templates…

Reply:
I would say don't let the perfect be the enemy of the good, and do what is needed to not have `!important`s on all the settings (looks like btn-primary is only thing affected, and I think .com/Lincs/Oxfordshire are all that is affected) and note other thing you'd like to do so they're not forgotten.